index.js

```diff
@@ -16,8 +16,132 @@ mongoose
     return Recipe.deleteMany()
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    return Recipe.create({
+      title: 'Pasta',
+      level: 'Easy Peasy',
+      ingredients: ['Pasta'],
+      cuisine: 'Italian',
+      dishType: 'main_course',
+      duration: 10,
+      creator: 'Us',
+    })
+  })
+  .then((result) => {
+    console.log(result)
+    return Recipe.insertMany([{
```

Comment:
That would make the code easier to read :) 

```suggestion
    return Recipe.insertMany(data)
```